jstreegrid.js

```diff
@@ -456,11 +459,25 @@
 					findDataCell(this.uniq,removeNodes,this._gridSettings.gridcols).remove();
 				}
 			}, this))
-			.on("hide_node.jstree", $.proxy(function (e, data) {
-				// remove out own data cells
-        findDataCell(this.uniq, data.node.id, this._gridSettings.gridcols).remove();
-				// hide any children
-        this._hide_grid(data.node);
+      .on("hide_node.jstree", $.proxy(function (e, data) {
```

Comment:
Should this be `on("hide_node.jstree")`? In this loop, you are **removing** css class `"jstree-grid-hidden"`. Is this supposed to be `.on("show_node.jstree")`?